rijsdijk2021reproducing/openreview/packages.tex

```diff
@@ -0,0 +1,26 @@
+\usepackage[final]
```

Comment:
What does this import do? Most probably the reason for the issue for compilation. Please remove this and re-try using `make` to compile

Reply:
That was under instructions of the openreview template. Now it is removed, and everything still works (better?).